docs/create/homebrew/index-homebrew.mdx

```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
 ![time series compaction](images/testredis6.png)
 
 
+</TabItem>
+<TabItem value="redis-on-macos">
+
+There are two ways to install Redis on Mac OS:
+
+- [Installing Redis from source](/create/from-source/)
+- Using Homebrew
+
+Homebrew is the easiest and most flexible way to install Redis on MacOS. It is a package management software for MacOS.
+It automates Redis installation process, making it quick and easy to add Redis to your system. 
+
+Follow the below steps to install Redis on Mac OS using brew service:
```

Comment:
```suggestion
Follow the below steps to install Redis on macOS using the Homebrew package manager:
```

Reply:
The term "brew service" and "Mac OS" has been added for SEO purpose.